package/boot/uboot-layerscape/files/fsl_ls1046a-frwy-sdboot-uEnv.txt

```diff
@@ -0,0 +1,9 @@
+fdtaddr=0x8f000000
+loadaddr=0x81000000
+fdt_high=0xffffffffffffffff
```

Comment:
Please fix fdt_high value for proper work fit image.

Reply:
Ok, thanks.

Reply:
My two boards (LS1012A-FRDM and LS1046A-RDB) won't start kernel from fit image with `fdt_high=0xffffffffffffffff`. Could You confirm, that it works for this boards?